src/org/thoughtcrime/securesms/ConversationItem.java

```diff
@@ -193,6 +194,7 @@ public void bind(@NonNull DcMsg                   messageRecord,
     setReactions(messageRecord);
     setFooter(messageRecord, locale);
     setQuote(messageRecord);
+    if (Util.isTouchExplorationEnabled(context)) setContentDescription();
```

Comment:
maybe a matter of taste, however, it is only a suggestion :)

```suggestion
    if (Util.isTouchExplorationEnabled(context)) {
      setContentDescription();
    }
```